### PR TITLE
Write Git and Jekyll documentation for newbies

### PR DESCRIPTION
**Pull request by Dalek-Sec**

### CODE REVIEW

**Comment by Dalek-Sec** [comment]
Resolves #26

File: README.md [hunk 1]
```diff
@@ -1,13 +1,98 @@
+# [Sparrow landing page repo](https://trysparrow.com)
```
Comment (dhanus) [inline README.md]: @Dalek-Sec Can you just say "Sparrow Landing Page" or "Sparrow Landing Page Repository"? If this is documentation for a total n00b, they might not know what a repo is yet.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,98 @@
...
+## Git usage
+If you're using Atom, you can use their GUI (graphical user interface) for git. This tutorial will use the git CLI (command line interface).
```
Comment (dhanus) [inline README.md]: @Dalek-Sec Change to: 
"We recommend using [Atom](LINK). You can use their ...  "

File: README.md [hunk 3]
```diff
@@ -1,13 +1,98 @@
...
+description: "<short description to show in previews. avoid anything that runs out of box in blog post listing>"
+preview: "[filename in path /assets/images/banners. will default to banner]"
+banner: "<filename in path /assets/images/banners>"
+permalink: "/blog/<post permenant url ending>"
+---
+
+```
+The blank line between front matter and content is required.
+#### Content
+Jekyll posts can be written in a mix of Markdown and HTML.
```
Comment (Dalek-Sec) [inline README.md]: Add link to Markdown tutorial/referrence

**Pull request merged**